metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/ExecutionTimeAthenaAggregator.java

```diff
@@ -25,13 +30,28 @@ Optional<ExecutionTimeMetric> createMetricFromQueryResultRow(QueryResultRow quer
         Optional<Double> min = getMinColumnValue(queryResultRow);
         Optional<Double> avg = getAvgColumnValue(queryResultRow);
         Optional<Double> max = getMaxColumnValue(queryResultRow);
+        Optional<Double> median = getMedianColumnValue(queryResultRow);
+        Optional<Double> percentile05th = get5thPercentileColumnValue(queryResultRow);
+        Optional<Double> percentile95th = get95thPercentileColumnValue(queryResultRow);
+
+        LOG.debug(" ");
+        LOG.debug("min: " + min.orElse(Double.NaN));
+        LOG.debug("05th: " + percentile05th.orElse(Double.NaN));
+        LOG.debug("avg: " + avg.orElse(Double.NaN));
+        LOG.debug("median: " + median.orElse(Double.NaN));
+        LOG.debug("95th: " + percentile95th.orElse(Double.NaN));
+        LOG.debug("max: " + max.orElse(Double.NaN));
+
         Optional<Integer> numberOfDataPoints = getCountColumnValue(queryResultRow);
         if (min.isPresent() && avg.isPresent() && max.isPresent() && numberOfDataPoints.isPresent()) {
```

Comment:
Should also confirm that the median/percentile Optionals (`median`, `percentile05th`, `percentile95th`) are present.